src/runtime.cpp

```diff
@@ -69,11 +71,23 @@ static size_t write_data(char* ptr, size_t size, size_t nmemb, void* userdata)
 
 static inline bool IsSpace(int ch)
 {
-    if (ch < -1 || ch > 255) {
-        return false;
+    constexpr int space = 0x20;           // space (0x20, ' ')
+    constexpr int form_feed = 0x0c;       // form feed (0x0c, '\f')
+    constexpr int line_feed = 0x0a;       // line feed (0x0a, '\n')
+    constexpr int carriage_return = 0x0d; // carriage return (0x0d, '\r')
+    constexpr int horizontal_tab = 0x09;  // horizontal tab (0x09, '\t')
+    constexpr int vertical_tab = 0x0b;    // vertical tab (0x0b, '\v')
+    switch (ch) {
+        case space:
+        case form_feed:
+        case line_feed:
+        case carriage_return:
+        case horizontal_tab:
+        case vertical_tab:
+            return true;
+        default:
+            return false;
```

Comment:
[isspace](https://en.cppreference.com/w/c/string/byte/isspace) has a cloud of UB around it. 
>The behavior is undefined if the value of ch is not representable as unsigned char and is not equal to EOF.

I'm not sure what "_not representable as unsigned char_" really means here. So I went ahead and did the simple thing I need to avoid the shenanigans.